.editorconfig

```diff
@@ -14,7 +14,7 @@ trim_trailing_whitespace = true
 indent_style = tab
 
 # 2 space indentation
-[*.{java,sh,json,xml}]
+[*.{sh,json,xml}]
```

Comment:
Why deleted `java`? 
`java` uses 2 spaces for indentation.

Reply:
When a new line is added in an expression or a method defination, we need to use 4 spaces for indentation. But this configuration will use 2 spaces. This will lead an error for style checking.